src/java.base/share/classes/com/sun/crypto/provider/CipherCore.java

```diff
@@ -363,7 +363,7 @@ private int getOutputSizeByOperation(int inputLen, boolean isDoFinal) {
         case GCM_MODE:
             if (isDoFinal) {
                 int tagLen = 0;
```

Comment:
Please remove the redundant initialization (this line is already modified relative to upstream).

Reply:
Removed